### PR TITLE
Minor fix to counter metric types description

### PR DESCRIPTION
**Pull request by jamtur01**
Signed-off-by: James Turnbull <james@lovedthanlost.net>

### CODE REVIEW

**Comment by jamtur01** [comment]
See #1035

File: content/docs/concepts/metric_types.md [hunk 1]
```diff
@@ -15,9 +15,8 @@ untyped time series. This may change in the future.
 
 A _counter_ is a cumulative metric that represents a single [monotonically 
 increasing counter](https://en.wikipedia.org/wiki/Monotonic_function) whose
-value can only increase or be reset to zero on restart. For example, you can
-use a counter to represent the number of requests served, tasks completed, or
-errors.
+value can only increase or be reset to zero on restart.  For example, do not use
```
Comment (brian-brazil) [inline content/docs/concepts/metric_types.md]: I don't think this was the change you meant to make

**Comment by jamtur01** [comment]
How odd. Let me fix that. Sorry!

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**